keyboards/keebio/quefrency/rev1/rev1.h

```diff
@@ -101,3 +101,27 @@
     { RE1, RE2, RE9, RE4, RE5, RE6, RE7, RE8 }, \
     { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RA9, RB9 } \
   }
+
+#define LAYOUT_MONAKA( \
```

Comment:
We usually follow the LAYOUT_all_lower_case_description_here formatting. 

```suggestion
#define LAYOUT_monaka( \
```